src/librbd/image/RemoveRequest.cc

```diff
@@ -621,6 +627,24 @@ void RemoveRequest<I>::handle_remove_v1_image(int r) {
   if (!m_old_format) {
     remove_v2_image();
   }
+
+}
+
+template<typename I>
+void RemoveRequest<I>::remove_image_cache() {
```

Comment:
An OpenStack Cinder controller running on nodeA will issue the librbd image remove request. However, that shared cache will not exist on the controller node but instead on hypervisor nodes nodeB - nodeZ.